tests/test_features.py

```diff
@@ -23,6 +25,10 @@ def _run_all_detectors(slither: Slither) -> None:
     slither.run_detectors()
 
 
+hardhat_available = shutil.which("hardhat") is not None
+
+
+@pytest.mark.skipif(not hardhat_available, reason="requires Hardhat and project setup")
```

Comment:
I'm trying to avoid the test failing locally as I imagine it does for most people. I have `npx` installed but not hardhat

Reply:
but then when crytic-compile runs `npx hardhat ...`, it will install it for you, won't it? 🤔

Reply:
The CI runs `npm install hardhat` likely because of this issue:
```
(slither) alpharush@macbook test_node_modules % slither .
'npx hardhat clean' running (wd: /Users/alpharush/tob/slither/tests/test_node_modules)
'npx' returned non-zero exit code 1
Error HH12: Trying to use a non-local installation of Hardhat, which is not supported.
```

Reply:
Thanks for figuring this out